### DISCUSSION TITLE
Bad german translation of BT-142-LotResult

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#991] [Ideas]
I would like to ask you to review the German translation of the BT-142-LotResult field. The German word "Preisträger" seems rather strange to me in this context.

This also applies to the translation of the GR-Winner group.

**Comment by YvesJo** [comment]
Hi,
Could you please specify the SDK version involved? In SDK 1.12, I find for the field BT-142-LotResult the following:
`<entry key="field|description|BT-142-LotResult">Angaben dazu, ob ein Gewinner ermittelt wurde.</entry>`
I confirm the observation for GR-Winner and will report it.

**Reply by JanEmde** [reply]
I am referring to the SDK 1.12. The "description" you quoted fits, but the name itself is: 

"field|name|BT-142-LotResult">Status der Preisträgerauswahl

**Reply by YvesJo** [reply]
Thanks for the details, I'll transfer them.
